install/gcp/gcp_installer.py

```diff
@@ -69,6 +69,9 @@ def run():
                         choices=['client', 'server'],
                         help='Type of the installation, '
                              'either client or server')
+    parser.add_argument('--target-project-id',
```

Comment:
a clearer name might be --inventoried-project

Reply:
Agreed and updated to --inventoried-project-id
I kept the -id to ensure the project id is used vs just the project name.